x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/add_domain/validation_step_panel.tsx

```diff
@@ -26,9 +33,14 @@ export const ValidationStepPanel: React.FC<ValidationStepPanelProps> = ({
   action,
 }) => {
   const showErrorMessage = step.state === 'invalid' || step.state === 'warning';
+  const styleOverride = showErrorMessage ? { paddingBottom: 0 } : {};
 
   return (
-    <EuiPanel hasShadow={false} color={domainValidationStateToPanelColor(step.state)}>
+    <EuiPanel
+      hasShadow={false}
+      color={domainValidationStateToPanelColor(step.state)}
+      style={styleOverride}
```

Comment:
Without this style override, the padding looked like this:

<img width="1305" alt="CleanShot 2021-09-09 at 14 17 17@2x" src="https://user-images.githubusercontent.com/809707/132691395-2c83c646-72eb-4f24-8a60-8823c4ea6171.png">